tests/lay-reflow-align-x-center/EPUB/content_001.xhtml

```diff
@@ -1,6 +1,12 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
   <head>
     <title>align-x-center in reflow</title>
+    <style>
+      html {
+        -epub-writing-mode:vertical-rl;
+        writing-mode:vertical-rl;
+      }
+    </style>
```

Comment:
Does this test need the text to be vertical? I worry about adding additional complexity when the test doesn't require it.

Reply:
"align-x-center" was added to respond to the request in Japanese vertical writing books.
Isn't this function necessary for horizontal writing?

The sample I made during the IDPF era was also written vertically.
https://github.com/toshiakikoike/epub-testsuite/tree/master/content/30/epub30-test-0130

https://github.com/w3c/epub-specs/issues/1380#issuecomment-719108924
As I wrote this issue, I think that only Voyager's BinB passes this test.

Reply:
If you want to leave it, that's fine, I just didn't want to overcomplicate the test.